models/DataObject/ClassDefinition/Data/Checkbox.php

```diff
@@ -242,11 +244,10 @@ public function getForWebserviceExport($object, $params = [])
     /**
      * converts data to be imported via webservices
      *
-     * @deprecated
      * @param mixed $value
      * @param null|Model\DataObject\AbstractObject $object
      * @param mixed $params
-     * @param Model\Webservice\IdMapperInterface|null $idMapper
+     * @param $idMapper
```

Comment:
This should be reverted. I think also the deprecated phpdocs and the isFilterable method.

Reply:
thx!